README.md

```diff
@@ -131,3 +132,27 @@ Please, go to [contributors page](https://github.com/fury-gl/fury/graphs/contrib
 We love contributions!
 
 You've discovered a bug or something else you want to change - excellent! Create an [issue](https://github.com/fury-gl/fury/issues/new)!
+
+# Citing
+
+If FURY is helping you during your work, [Please, cite us!](https://doi.org/10.21105/joss.03384) By citing us, you will participate to the FURY growth!
```

Comment:
If you are using FURY in your work then do cite [this paper](https://doi.org/10.21105/joss.03384). By citing FURY you are helping sustain the FURY ecosystem.